ast/fuzz.go

```diff
@@ -9,6 +9,7 @@ import (
 var blacklist = []string{
 	"{{{{{", // nested { and [ cause the parse time to explode
 	"[[[[[",
+	"[{{[{{{{[{{",
```

Comment:
We need to generalize this a bit and just check for repeated opening brackets/braces.

```
(master)torin:~/go/src/github.com/open-policy-agent/opa$ time opa eval --metrics '{{{{{{'
1 error occurred: 1:7: rego_parse_error: no match found
	{{{{{{
	     ^

real	0m0.278s
user	0m0.324s
sys	0m0.021s
(master)torin:~/go/src/github.com/open-policy-agent/opa$ time opa eval --metrics '{{{{{{{'
1 error occurred: 1:8: rego_parse_error: no match found
	{{{{{{{
	      ^

real	0m1.002s
user	0m1.263s
sys	0m0.033s
(master)torin:~/go/src/github.com/open-policy-agent/opa$ time opa eval --metrics '{{{{{{{{'
1 error occurred: 1:9: rego_parse_error: no match found
	{{{{{{{{
	       ^

real	0m3.940s
user	0m4.734s
sys	0m0.117s
(master)torin:~/go/src/github.com/open-policy-agent/opa$ time opa eval --metrics '{{{{{{{{{'
1 error occurred: 1:10: rego_parse_error: no match found
	{{{{{{{{{
	        ^

real	0m16.022s
user	0m18.718s
sys	0m0.468s
```

6 seems like a decent cut-off.